ruby_on_rails/rspec.md

```diff
@@ -126,11 +126,21 @@ Commit and push your changes! :tada:
 
 ## Verify
 
+Check that you see a green page in each app.
+
+### Heroku
```

Comment:
this check should be done also for deplo.io, no?

Reply:
Yes, it is also described in the `### Deploio` section. The difference is that the application urls are generated and can be retrieved using the nctl cli.